src/main/app/src/app/shared/edit/edit-tekst/edit-tekst.component.less

```diff
@@ -0,0 +1,5 @@
+/*
+ * SPDX-FileCopyrightText: 2021 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
```

Comment:
Nu deze file leeg is is ie dan nog wel nodig?

Reply:
OK